### PR TITLE
LPS-73461 Make shims for Node.js globals and modules available to the Javascript code run in the portal

### PR DESCRIPTION
**Pull request by izaera**

### CODE REVIEW

File: modules/apps/foundation/frontend-js/frontend-js-node-shims/build.gradle [hunk 1]
```diff
@@ -0,0 +1,5 @@
+apply plugin: "com.liferay.node"
+
+classes {
+	dependsOn npmRunBuild
```
Comment (jbalsas) [inline modules/apps/foundation/frontend-js/frontend-js-node-shims/build.gradle]: Is this really needed? Should be automatic...

Comment (izaera) [inline modules/apps/foundation/frontend-js/frontend-js-node-shims/build.gradle]: Yes, it is automatic. It didn't work for gulp, but it is automatic for NPM :-o

Comment (izaera) [inline modules/apps/foundation/frontend-js/frontend-js-node-shims/build.gradle]: I will remove the dependency and add the npm-shrinkwrap.json 👍

**Comment by izaera** [comment]
ci:stop

**Comment by izaera** [comment]
ci:reopen
ci:stop
ci:retest

**Comment by izaera** [comment]
ci:retest

**Comment by izaera** [comment]
ci:retest

**Comment by izaera** [comment]
ci:retest

**Comment by izaera** [comment]
ci:retest

**Comment by jbalsas** [comment]
ci:reopen

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/jbalsas/liferay-portal/pull/778. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jbalsas** [comment]
Pull request submitted to https://github.com/izaera/liferay-portal/pull/10. [See changes here.](https://github.com/jbalsas/liferay-portal/compare/b548e55~4...b548e55) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**